arm7/lib/include/PM.h

```diff
@@ -0,0 +1,9 @@
+#ifndef GUARD_PM_H
+#define GUARD_PM_H
+
+#include "nitro/types.h"
+
+void PMi_SetControl(int ctrl);
```

Comment:
does s32 or u32 work here?

Reply:
Actually, I've realized both are probably wrong. I replaced it with u8.